File: test_archives/py_tests/batch2/test_function9.py
from functions.function9 import function9
import pytest

def test_function9_1():
    assert function9(1, 2.5, "foo", [1,2,3]) == 12.5

def test_function9_2():
    assert function9(4, 7.2, "bar", [3,4,5]) == 26.2

File: test_archives/cpp_tests/batch2/test_function9.cpp
#include "../catch.hpp"
#include "../functions/function9.cpp"

TEST_CASE( "Testing function9 for two cases", "[function9]" ) {
    REQUIRE( function9(1, 2.5, "foo", {1,2,3}) == 12.5 );
    REQUIRE( function9(4, 7.2, "bar", {3,4,5}) == 26.2 );
}